str0m/src/media/mline.rs

```diff
@@ -399,7 +406,21 @@ impl MLine {
                 None => continue,
             };
 
+            let from = now.checked_sub(Duration::from_secs(1)).unwrap_or(now);
+            let bytes_transmitted = self.bytes_transmitted.sum_from(from);
+            let bytes_retransmitted = self.bytes_retransmitted.sum_from(from);
+            let ratio =
+                bytes_retransmitted as f32 / (bytes_retransmitted + bytes_transmitted) as f32;
+            let ratio = if ratio.is_finite() { ratio } else { 0_f32 };
+
+            if ratio > 0.15_f32 {
```

Comment:
Do we have a reason for 0.15 here?

Reply:
Not a strong one. I tested with high values of loss and congestion and 0.1 was fine, only rarely there was some small image corruption. Then I increased to 0.15 to be safer.

Reply:
Alright. Let's try and see!